addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -612,7 +612,6 @@
         <field name="res_model">hr.leave</field>
         <field name="binding_model_id" eval="False"/>
         <field name="view_mode">tree,kanban,form,calendar,activity</field>
-        <field name="context">{'form_view_ref': 'hr_holidays.hr_leave_view_form_manager'}</field>
```

Comment:
can't you add it in view_ids instead of only removing it?

Reply:
Indeed, should be good!

Reply:
It should be:
```
<field name="context">{}</field>
```
Otherwise people won't have the fix appleid.

Reply:
Oops forgot to add it, should be good this time